Gaijinizer patch/Troops.txt

```diff
@@ -7279,18 +7287,20 @@ It will be broken down with the scale ......! !#MTLed
 ベタな方法だけど、アンタみたいに硬いウロコで
 防御する悪魔には効果テキメンだろ！！
 > CONTEXT: Troops/170/pages/0/36/Dialogue/hyakki
-"After the heat, suddenly cool"
-It's a solid method, but it's a hard scale like
-you. It is an effective text for the demon to
-Defend! !#MTLed
+Rapid heating followed by rapid cooling.
+It's unimaginative, but it's effective against
+a demon like you who can use her scales as such a
+solid defense!!#Ron
```

Comment:
As mentioned in our talks, a direct translation here probably wouldn't work, let's localize it a bit:
>_**It's nothing fancy, but it's effective against
a demon like you who can use her scales as such a
solid defense!!**_